content/events/devcon24/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "[Upcoming Event] Espressif DevCon24"
+date: 2024-06-11
+draft: false
+description: "Espressif Systems Developer Conference 24"
+tags: ["event", "announcement", "devcon"]
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## DevCon24
+
+Espressif DevCon24, the third edition of Developers conference made by Developers for Developers!
+
+{{< alert icon="eye">}}
+**The DevCon24, will take place online from September 3rd to 5th 13.00 to 18.00 CEST.**
+{{< /alert >}}
+
+During the three-day online conference, we will provide you with nearly 30 talks created by Espressif technical experts and its partners, covering a wide range of topics, including Thread, Matter, ESP-IDF, Arduino Core for ESP32, ESP-Mesh-Lite + ESP RainMaker Cloud solution, RUST and more.
+
+> *Full program will be shared later.*
+
+For more information, please visit [DevCon24](devcon.espressif.com)
+
+Event is for free and will be streamed on [Espressif's YouTube channel](https://www.youtube.com/channel/UCDBWNF7CJ2U5eLGT7o3rKog).
+
+You can watch here, the DevCon23 keynote from our CEO, Teo Swee Ann.
+
+{{< youtube mR3gUNXMEsM >}}
+
+---
+
+## About Espressif Systems
+
+Espressif Systems focuses on the R&D and production of wireless communication chips and modules and the development of operating system software. Established in 2008 with headquarters in Shanghai, the company has offices in Brazil, the Czech Republic, India and Singapore.
```

Comment:
Is it a standard description? The text can be improved. Too many "and" here. What is meant by operating system software?

Reply:
I'll remove it. This is from the LinkedIn event post.